test/models/user_test.rb

```diff
@@ -1,7 +1,18 @@
 require 'test_helper'
 
 class UserTest < ActiveSupport::TestCase
```

Comment:
- [ ] Great work on making the validations and tests but you are supposed to use RSpec to write all your tests/specs of models and validation :thinking: but you have used Rails Testing :fearful: Kindly use RSpec to test your models and validations to make your project follow the requirements :blush: